src/main/java/com/iridium/iridiumskyblock/configs/Biomes.java

```diff
@@ -0,0 +1,186 @@
+package com.iridium.iridiumskyblock.configs;
+
+import com.google.common.collect.ImmutableMap;
+import com.iridium.iridiumcore.Item;
+import com.iridium.iridiumcore.dependencies.fasterxml.annotation.JsonIgnore;
+import com.iridium.iridiumcore.dependencies.xseries.XBiome;
+import com.iridium.iridiumcore.dependencies.xseries.XMaterial;
+import com.iridium.iridiumcore.dependencies.xseries.XSound;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+
+import java.util.*;
+
+public class Biomes {
+
+    public Map<String, BiomeCategory> categories;
+    public Map<String, List<BiomeItem>> items;
+    public String buyPriceLore;
+    public String notPurchasableLore;
+    public boolean abbreviatePrices;
+    public XSound failSound;
+    public XSound successSound;
+    public List<String> biomeItemLore;
+
+    public Biomes() {
+        this("&c");
+    }
+
+    public Biomes(String color) {
```

Comment:
This color thing was done before, as a way for plugins (Skyblock) to specify a color in IridiumTeams

Since this is Skyblock only, we dont really need to do that